src/tests/discord/events/message/index.test.ts

```diff
@@ -70,7 +80,26 @@ describe('message handler', () => {
 
     await message(genericMessage);
     expect(reply).toBeCalledTimes(1);
-    expect(reply).toBeCalledWith("That isn't a command I understand. Try replying with `!help` to see the full list of things I can help with!");
+    expect(reply).toBeCalledWith(stringDictionary.botCantUnderstand);
+  });
+
+  it('successfully responds to !H requests', async () => {
+    const reply = jest.fn();
+    const helpMessage = makeDiscordMessage({
+      reply,
+      content: '!H',
+      author: {
+        id: mockDiscordContext.id,
+      },
+      channel: {
+        type: 'dm',
+      },
+    });
+
+    await message(helpMessage);
+    expect(mockHelpHandler).toBeCalledTimes(1);
+    const messageArg = mockHelpHandler.mock.calls[0][0];
+    expect(messageArg).toEqual(helpMessage);
```

Comment:
This isn't needed, since this is actually testing that your test passed the correct object in.

```suggestion
```

Reply:
Looks like this pattern is in a couple more tests, make sure you remove those as well